integrations/Woopra/browser.js

```diff
@@ -0,0 +1,128 @@
+/* eslint-disable func-names */
+/* eslint-disable no-unused-expressions */
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable class-methods-use-this */
+import logger from "../../utils/logUtil";
+
+import { NAME } from "./constants";
+
+class Woopra {
+  constructor(config) {
+    this.projectName = config.projectName;
+    this.name = NAME;
+    this.cookieName = config.cookieName;
+    this.cookiePath = config.cookiePath;
+    this.cookieDomain = config.cookieDomain;
+    this.clickTracking = config.clickTracking;
+    this.downloadTracking = config.downloadTracking;
+    this.hideCampaign = config.hideCampaign;
+    this.idleTimeout = config.idleTimeout;
+    this.ignoreQueryUrl = config.ignoreQueryUrl;
+    this.outgoingIgnoreSubdomain = config.outgoingIgnoreSubdomain;
+    this.outgoingTracking = config.outgoingTracking;
+  }
+
+  loadScript() {
+    !(function () {
+      var t,
+        o,
+        c,
+        e = window,
+        n = document,
+        r = arguments,
+        a = "script",
+        i = [
+          "call",
+          "cancelAction",
+          "config",
+          "identify",
+          "push",
+          "track",
+          "trackClick",
+          "trackForm",
+          "update",
+          "visit",
+        ],
+        s = function () {
+          var t,
+            o = this,
+            c = function (t) {
+              o[t] = function () {
+                return (
+                  o._e.push(
+                    [t].concat(Array.prototype.slice.call(arguments, 0))
+                  ),
+                  o
+                );
+              };
+            };
+          for (o._e = [], t = 0; t < i.length; t++) c(i[t]);
+        };
+      for (e.__woo = e.__woo || {}, t = 0; t < r.length; t++)
+        e.__woo[r[t]] = e[r[t]] = e[r[t]] || new s();
+      ((o = n.createElement(a)).async = 1),
```

Comment:
The dataset attribute is not set similarly to other integrations.

Reply:
I think this is what the woopra script suggests. Am I missing something @anantjain45823 ?

Reply:
Yeah this is the Woopra script.
Ref: https://docs.woopra.com/reference/the-javascript-snippet